src/reanimated2/layoutReanimation/web/transition/Fading.web.ts

```diff
@@ -0,0 +1,55 @@
+import { parseAnimationObjectToKeyframe } from '../animationParser';
+import type { TransitionData } from '../animationParser';
+
+export function FadingTransition(name: string, transitionData: TransitionData) {
+  const { translateX, translateY, scaleX, scaleY } = transitionData;
+
+  const fadingTransition = {
+    name: name,
+    style: {
+      0: {
+        opacity: 1,
+        transform: [
+          {
+            translateX: `${translateX}px`,
+            translateY: `${translateY}px`,
+            scale: `${scaleX}, ${scaleY}`,
+          },
+        ],
+      },
+      20: {
+        opacity: 0,
+        transform: [
+          {
+            translateX: `${translateX}px`,
+            translateY: `${translateY}px`,
+            scale: `${scaleX}, ${scaleY}`,
+          },
+        ],
+      },
+      60: {
+        opacity: 0,
+        transform: [
+          {
+            translateX: '0px',
+            translateY: '0px',
+            scale: `1,1`,
```

Comment:
Sometimes there's a space after comma (e.g. `${scaleX}, ${scaleY}`) and sometimes there's no space (e.g. `1,1`). Can we unify it across all files?

Reply:
Done in [5aafc4d](https://github.com/software-mansion/react-native-reanimated/pull/4805/commits/5aafc4d8a2c09e39ccdf76d490e7446b27320ae2).